website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -65,30 +65,50 @@ There are two ways to create RDS and S3 resources before you deploy the Kubeflow
 
 This setup performs all the manual steps in an automated fashion.  
 
-The script takes care of creating the S3 bucket, creating the S3 Secrets using the Secrets manager, setting up the RDS database, and creating the RDS Secret using the Secrets manager. The script also edits the required configuration files for Kubeflow Pipelines to be properly configured for the RDS database during Kubeflow installation. The script also handles cases where the resources already exist. In this case, the script will simply skip the step.
+The script takes care of creating the S3 bucket, creating the S3 Secrets if using static credentials using the Secrets manager or setting up IRSA to access to S3, setting up the RDS database, and creating the RDS Secret using the Secrets manager. The script also edits the required configuration files for Kubeflow Pipelines to be properly configured for the RDS database during Kubeflow installation. The script also handles cases where the resources already exist. In this case, the script will simply skip the step.
```

Comment:
nit: Since irsa is the default, shouldn't it be included first `setting up IRSA to access to S3 or creating the S3 Secrets if using static credentials using the Secrets manager`